ironic.conf

```diff
@@ -21,8 +21,7 @@ deploy_logs_collect = always
 deploy_logs_local_path = /shared/log/ironic/deploy
 
 [conductor]
-automated_clean = true
-bootloader = /httpboot/uefi_esp.img
```

Comment:
Was this removed intentionally?

Reply:
Yes, intentional. It controls automatic move through cleaning from provide verb to available state.

Reply:
Sorry was referring to the bootloader line, looks like it get fixed when resolving merge conflicts.